test/test.js

```diff
@@ -81,10 +80,10 @@ describe('Generate APIs and models with WSDLs containing ', function() {
           };
 
           var code = helper.generateRemoteMethods(apiData);
-          //console.log(code);
+          // console.log(code);
```

Comment:
I purposely left it commented out console.log statements for easy debugging purpose for future failures if any... I can remove it.